tests/test_makepackage.py

```diff
@@ -1,102 +1,77 @@
+        PY = "python"
     else:
-        return windows_object
+        PY = "python3"
 
+    subprocess.run(f"{PY} -m venv venv-pkgWithCLI", shell=True, cwd=tmp_path)
+    subprocess.run(f"makepackage pkgWithCLI --cli", shell=True, cwd=tmp_path)
 
-def test_folders_no_CLI(testing_path: Path):
-    assert_sys(
-        (testing_path / "venv-pkgNoCLI").exists(),
-        (testing_path / "venv-pkgNoCLIWin").exists(),
-    )
-    assert_sys(
-        (testing_path / "pkgNoCLI").exists(),
-        (testing_path / "pkgNoCLIWin").exists(),
-    )
-    assert_sys(
-        (testing_path / "pkgNoCLI" / "pkgNoCLI").exists(),
-        (testing_path / "pkgNoCLIWin" / "pkgNoCLIWin").exists(),
-    )
-    assert_sys(
-        (testing_path / "pkgNoCLI" / "tests").exists(),
-        (testing_path / "pkgNoCLIWin" / "tests").exists(),
-    )
+    assert (tmp_path / "venv-pkgWithCLI").exists()
+    assert (tmp_path / "pkgWithCLI").exists()
+    assert (tmp_path / "pkgWithCLI" / "pkgWithCLI").exists()
+    assert (tmp_path / "pkgWithCLI" / "tests").exists()
 
 
-def test_folders_with_CLI(testing_path: Path):
-    assert_sys(
-        (testing_path / "venv-pkgWithCLI").exists(),
-        (testing_path / "venv-pkgWithCLIWin").exists(),
-    )
-    assert_sys(
-        (testing_path / "pkgWithCLI").exists(),
-        (testing_path / "pkgWithCLIWin").exists(),
-    )
-    assert_sys(
-        (testing_path / "pkgWithCLI" / "pkgWithCLI").exists(),
-        (testing_path / "pkgWithCLIWin" / "pkgWithCLIWin").exists(),
-    )
-    assert_sys(
-        (testing_path / "pkgWithCLI" / "tests").exists(),
-        (testing_path / "pkgWithCLIWin" / "tests").exists(),
-    )
+def test_pkg_files_no_CLI(tmp_path: Path, files_no_CLI: Dict[str, List[str]]):
+    if platform.system() == "Windows":
+        PY = "python"
+    else:
+        PY = "python3"
+
+    subprocess.run(f"{PY} -m venv venv-pkgNoCLI", shell=True, cwd=tmp_path)
+    subprocess.run(f"makepackage pkgNoCLI", shell=True, cwd=tmp_path)
 
+    files = files_no_CLI
 
-def test_pkg_files_no_CLI(
-    testing_path: Path, files_no_CLI: Dict[str, List[str]]
-):
-    root_dir = choose_object_for_sys(
-        testing_path / "pkgNoCLI", testing_path / "pkgNoCLIWin"
+    assert all((tmp_path / "pkgNoCLI" / file).exists() for file in files["root"])
+    assert all(
+        (tmp_path / "pkgNoCLI" / "pkgNoCLI" / file).exists() for file in files["src"]
     )
-    src_dir = choose_object_for_sys(
-        root_dir / "pkgNoCLI", root_dir / "pkgNoCLIWin"
+    assert all(
+        (tmp_path / "pkgNoCLI" / "tests" / file).exists() for file in files["test"]
     )
-    tests_dir = root_dir / "tests"
 
-    files = files_no_CLI
-    assert all((root_dir / file).exists() for file in files["root"])
-    assert all((src_dir / file).exists() for file in files["src"])
-    assert all((tests_dir / file).exists() for file in files["test"])
 
+def test_pkg_files_with_CLI(tmp_path: Path, files_with_CLI: Dict[str, List[str]]):
```

Comment:
In this version, it does not make much sense to write two test functions, one for folders and one for files, as they create the same environment. Hence, please join them. So, we will have two test functions: one without and another with CLI.